packages/scandipwa/src/component/Overlay/Overlay.container.js

```diff
@@ -24,4 +24,5 @@ export const mapStateToProps = (state) => ({
 // eslint-disable-next-line no-unused-vars
 export const mapDispatchToProps = (dispatch) => ({});
 
-export default connect(mapStateToProps, mapDispatchToProps)(OverlayComponent);
+// eslint-disable-next-line @scandipwa/scandipwa-guidelines/always-both-mappings
```

Comment:
Why is this disabled? Is there a bug in this rule?

Reply:
@yeegor 
Looks so, exactly 2 arguments are required for rule not to fail:
https://github.com/scandipwa/eslint/blob/master/lib/rules/always-both-mappings.js#L20